modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -278,6 +286,8 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
 
         virtual Ptr<BackendNode> initInfEngine(const std::vector<Ptr<BackendWrapper> > &inputs);
 
+        virtual Ptr<BackendNode> initNgraph(const std::vector<Ptr<BackendWrapper> > &inputs, const std::vector<Ptr<BackendNode> >& nodes);
```

Comment:
Lets keep it here for this first iteration (keep code separate for easy regressions investigations)